graphene/src/quad.rs

```diff
@@ -59,4 +60,23 @@ impl Quad {
         assert!(index_ < 4);
         unsafe { from_glib_none(ffi::graphene_quad_get_point(self.to_glib_none().0, index_)) }
     }
+
+    pub fn points(&self) -> [Point; 4] {
+        unsafe {
+            [
+                from_glib_none(&self.0.points[0] as *const _),
+                from_glib_none(&self.0.points[1] as *const _),
+                from_glib_none(&self.0.points[2] as *const _),
+                from_glib_none(&self.0.points[3] as *const _),
```

Comment:
Why not just `&*(&self.0.points as *const [ffi::graphene_point_t; 4] as *const [Point; 4])`

Reply:
I wasn't aware of this, cleaned up a few cases like this one as well